src/main/groovy/com/groovycoder/spockdockerextension/TestcontainersMethodInterceptor.groovy

```diff
@@ -74,15 +72,15 @@ class TestcontainersMethodInterceptor extends AbstractMethodInterceptor {
     }
 
     private static void startContainers(List<FieldInfo> containers, IMethodInvocation invocation) {
-
         containers.each { FieldInfo f ->
             GenericContainer container = readContainerFromField(f, invocation)
-            container.start()
+            if(!container.isRunning()){
```

Comment:
Stop method in `GenericContainer` already has a protection against double stopping